go.mod

```diff
@@ -130,6 +130,8 @@ replace github.com/zondax/hid => github.com/zondax/hid v0.9.1-0.20220302062450-5
 
 replace github.com/99designs/keyring => github.com/cosmos/keyring v1.1.7-0.20210622111912-ef00f8ac3d76
 
+replace github.com/confio/ics23/go => github.com/agoric-labs/cosmos-sdk/ics23/go v0.8.0-alpha.agoric.1
```

Comment:
was this supposed to be v0.8.8?

Reply:
Nope - both cosmos/cosmos-sdk and agoric-labs/cosmos-sdk only use `v0.8.0`.

Reply:
ah either will work.  was confused since you were going to tag v0.8.8 per https://github.com/agoric-labs/cosmos-sdk/pull/264